app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (
```

Comment:
The imports can probably fit into one line.

Reply:
I like how it looks and don't think this is an error